src/utils/getUserByID.tsx

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/types';
+import usersFromServer from '../api/users';
+
+export function getUserById(id: number): User {
+  return usersFromServer.find(user => user.id === id)!;
```

Comment:
[CHECKLIST ITEM 5] You're importing and using data directly in a non-root component. It's recommended to prepare all data in one place (App component) and pass it to child components. This approach will make your code more maintainable and predictable. Try to refactor your code to follow this recommendation. For example, you could pass the `usersFromServer` as an argument to the `getUserById` function.